python/cudf/cudf/tests/test_parquet.py

```diff
@@ -754,22 +754,6 @@ def test_parquet_reader_arrow_nativefile(parquet_path_or_buf):
     assert_eq(expect, got)
 
 
-def test_parquet_reader_use_python_file_object(parquet_path_or_buf):
-    # Check that the non-default `use_python_file_object=True`
-    # option works as expected
-    expect = cudf.read_parquet(parquet_path_or_buf("filepath"))
-    fs, _, paths = get_fs_token_paths(parquet_path_or_buf("filepath"))
-
-    # Pass open fsspec file
-    with fs.open(paths[0], mode="rb") as fil:
-        got1 = cudf.read_parquet(fil, use_python_file_object=True)
-    assert_eq(expect, got1)
-
-    # Pass path only
-    got2 = cudf.read_parquet(paths[0], use_python_file_object=True)
-    assert_eq(expect, got2)
```

Comment:
This behavior is now the default (and is already covered in many other tests).

Reply:
IIUC we're planning on deprecating and removing the ability for the user to use `use_python_file_object=False`, correct? If so perhaps we should leave the test in with that option until it is actually removed. There maybe should also be an issue to track

Reply:
I agree - I'll add explicit test coverage for `use_python_file_object=False`